AmplifyPlugins/DataStore/Sources/AWSDataStorePlugin/Sync/SubscriptionSync/IncomingAsyncSubscriptionEventPublisher.swift

```diff
@@ -202,15 +202,15 @@ final class IncomingAsyncSubscriptionEventPublisher: AmplifyCancellable {
         let request: GraphQLRequest<Payload>
         if modelSchema.hasAuthenticationRules,
             let _ = auth,
-            let tokenString = try? await awsAuthService.getUserPoolAccessToken(),
+            let tokenString = try? await awsAuthService.getLatestAuthToken(),
```

Comment:
revert changes here as well, to getUserPoolAccessToken() ?

Reply:
Change the protocol and the build should fail and help you cascade the required changes

Reply:
take this PR for example https://github.com/aws-amplify/amplify-ios/pull/2101 it only changes the protocol, then try to build Amplify, APIPlugin, GraphQL Lambda Auth integration tests, DataStorePlugin. From there you should find the places which conform to the new protocol method